tekton/ci/templates/catalog-template.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: TriggerTemplate
+metadata:
+  name: tekton-catalog-ci-pipeline
+  namespace: tektonci
+  annotations:
+    triggers.tekton.dev/old-escape-quotes: "true"
+spec:
+  params:
+    - name: buildUUID
+      description: UUID used to track a CI Pipeline Run in logs
+    - name: package
+      description: org/repo
+    - name: pullRequestNumber
+      description: The pullRequestNumber
+    - name: pullRequestUrl
+      description: The HTML URL for the pull request
+    - name: gitRepository
+      description: The git repository that hosts context and Dockerfile
+    - name: gitRevision
+      description: The Git revision to be used.
+    - name: gitCloneDepth
+      description: Number of commits in the change + 1
+    - name: gitHubCommand
+      description: |
+        The GitHub command that was used a trigger. This is only available when
+        this template is triggered by a comment. The default value is for the
+        case of a pull_request event.
+      default: ""
+    - name: labels
+      description: List of labels currently on the Pull Request
+  resourcetemplates:
+    - apiVersion: tekton.dev/v1beta1
+      kind: PipelineRun
+      metadata:
+        generateName: catalog-catlin-run-
+        labels:
+          prow.k8s.io/build-id: $(tt.params.buildUUID)
+          tekton.dev/check-name: pull-catalog-catlin-lint
+          tekton.dev/kind: ci
+          tekton.dev/pr-number: $(tt.params.pullRequestNumber)
+        annotations:
+          tekton.dev/gitRevision: "$(tt.params.gitRevision)"
+          tekton.dev/gitURL: "$(tt.params.gitRepository)"
+      spec:
+        serviceAccountName: tekton-ci-jobs
+        pipelineRef:
+          name: catlin-linter
+        workspaces:
+          - name: store-changed-files
```

Comment:
I don't think you need a PVC to share files between two steps? An emptydir should be enough

Reply:
:man_facepalming: I forgot to change this. I'll fix this :sweat_smile: